packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.scss

```diff
@@ -4,40 +4,43 @@
 *--------------------------------------------------------------------------------------------*/
 
 .presentation-instance-filter-dialog {
-  min-width: 50vw;
+  .presentation-instance-filter-dialog-content-container {
+    min-height: calc(var(--iui-size-2xl) * 5 + var( --iui-size-xs));
+    min-width: calc(var(--iui-size-2xl) * 8);
 
-  .presentation-instance-filter-title {
-    margin-bottom: 0;
-    padding: 6px;
-  }
+    .presentation-instance-filter-title {
+      margin-bottom: 0;
+      padding: 6px;
+    }
 
-  .presentation-instance-filter-content {
-    max-height: 80vh;
-    overflow: hidden;
+    .presentation-instance-filter-content {
+      max-height: 80vh;
+      overflow: hidden;
 
-    .presentation-property-filter-builder {
-      overflow: scroll;
-      max-height: 70vh;
+      .presentation-property-filter-builder {
+        overflow: scroll;
+        height: calc(100% - var(--iui-size-xl));
```

Comment:
Looks like this does not divide height equally between `Class selector` and `Filter builder`. It seems that `Class selector` is slightly bigger than `--iui-size-xl`. 

I played with it locally and it looks okay with `Class selector: height: var(--iui-size-2xl), padding-top: 10px` (moved padding from bottom to the top). 
With this style applied `FilterBuilder` can use `height: calc(100% - var(--iui-size-2xl)`. 
What do you think?

Reply:
fixed